### PR TITLE
styling update

### PR DESCRIPTION
**Pull request by KristePe**
Some styling updates, but not finished.

### CODE REVIEW

File: frontend/src/components/inputs/Inputs.css [hunk 1]
```diff
@@ -16,9 +33,27 @@
...
   display: block;
-  font: 1rem "Fira Sans", sans-serif;
-  min-width: 10rem;
+  font: 0.8rem "Fira Sans", sans-serif;
+  
+
+  /* min-width: 10rem; */
+  
+/* Experimental code */
+border-radius: 0;
```
Comment (VaidaLinkuviene) [inline frontend/src/components/inputs/Inputs.css:43]: Format

File: frontend/src/components/inputs/Inputs.css [hunk 2]
```diff
@@ -62,7 +116,7 @@
 
 .date-input:valid + .date-picker::before {
   content: "Select a date";
-  color: black;
+  /* color: black; */
```
Comment (VaidaLinkuviene) [inline frontend/src/components/inputs/Inputs.css:106]: If it is working, let rid off all comments

File: frontend/src/components/sideCanvas/SideCanvas.css [hunk 3]
```diff
@@ -14,21 +19,78 @@
...
+
+.MainWindow {
+  height: 100vh;
+}
+
+.SideCanvas.light {
+  background: #a7aab1;
+  color: #333333;
+  transition: 0.3s;
+
```
Comment (VaidaLinkuviene) [inline frontend/src/components/sideCanvas/SideCanvas.css]: Remove gaps

**Comment by KristePe** [comment]
Fixed code according comments.

File: frontend/src/components/Balance/Balance.css [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
+    justify-content: space-around;
+    padding: 10px;
 
 }
 
 .balance-title{
-    font-size: 30px;
+    font-size: 18px;
     font-weight: 600;
 }
```
Comment (VaidaLinkuviene) [inline frontend/src/components/Balance/Balance.css]: add one more line

File: frontend/src/components/header/Header.css [hunk 5]
```diff
@@ -1,20 +1,23 @@
...
+  height: 2rem;
   display: flex;
   justify-content: space-between;
-  z-index: 1;
+  z-index: 100;
   font-family: "Pacifico", cursive;
   align-items: center;
-  padding: 0 10px 0 10px;
+  padding: 10px;
+
```
Comment (VaidaLinkuviene) [inline frontend/src/components/header/Header.css]: delete gap

File: frontend/src/components/header/Header.css [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
+  top: 0;
 }
 
 header {
   display: flex;
   justify-items: space-between;
   align-items: center;
   justify-content: center;
+
```
Comment (VaidaLinkuviene) [inline frontend/src/components/header/Header.css]: delete gap

File: frontend/src/components/header/Header.css [hunk 7]
```diff
@@ -56,19 +59,20 @@ header {
   transition: 0.3s;
 }
 
-h1 {
-  color: rgb(100, 27, 169);
-}
+
```
Comment (VaidaLinkuviene) [inline frontend/src/components/header/Header.css]: delete gap

File: frontend/src/components/header/Header.js [hunk 8]
```diff
@@ -12,6 +12,8 @@ const Header = () => {
       <div className="header-wrapper">
         <div>
           <h2>My Expense Tracker</h2>{" "}
+
```
Comment (VaidaLinkuviene) [inline frontend/src/components/header/Header.js]: delete gap

**Review by VaidaLinkuviene — COMMENTED** [review]
And do not forget Ctrl A +K + F to format the code

**Comment by KristePe** [comment]
Fixed!

**Review by VaidaLinkuviene — APPROVED** [review]
Great!

**Pull request merged**